src/juicenet/nyuu.py

```diff
@@ -51,6 +53,7 @@ def __init__(
         scope: str,
         debug: bool,
         bdmv_naming: bool,
+        meta: Optional[list[str]],
```

Comment:
```suggestion
        meta: Optional[list[str]] = None,
```
This should set a default value to keep it backward-compatible so that existing calls without `meta` still work.

Reply:
`Optional[list[str]]` is only a *type hint*. It has no effect at runtime.
As the [typing docs](https://docs.python.org/3/library/typing.html) note:

> The Python runtime does not enforce function and variable type annotations. They can be used by third party tools such as [type checkers](https://docs.python.org/3/glossary.html#term-static-type-checker), IDEs, linters, etc.

For example:

```pycon
>>> from typing import Optional
>>> def foobar(a: int, b: Optional[int]):
...     print("Type of a is:", type(a).__name__)
...     print("Type of b is:", type(b).__name__)
...
>>> foobar("hello", "world")  # Hints are ignored at runtime
Type of a is: str
Type of b is: str

>>> foobar("hello")  # Missing the second argument is still a runtime error
Traceback (most recent call last):
  ...
TypeError: foobar() missing 1 required positional argument: 'b'

>>> # Use a default if you truly want 'b' to be optional:
>>> def foobar(a: int, b: Optional[int] = None):
...     print("Type of a is:", type(a).__name__)
...     print("Type of b is:", type(b).__name__)
...
>>> foobar("hello")
Type of a is: str
Type of b is: NoneType
```